vhdbuilder/packer/imagecustomizer/azlosguard/azlosguard.yml

```diff
@@ -0,0 +1,316 @@
+      permissions: 600
+    # secure-tls-bootstrap
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/secure-tls-bootstrap.service
+      destination: /etc/systemd/system/secure-tls-bootstrap.service
+      permissions: 600
+    # aks log collector
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector.sh
+      destination: /opt/azure/containers/aks-log-collector.sh
+      permissions: 755
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector-send.py
+      destination: /opt/azure/containers/aks-log-collector-send.py
+      permissions: 755
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector.service
+      destination: /etc/systemd/system/aks-log-collector.service
+      permissions: 600
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector.slice
+      destination: /etc/systemd/system/aks-log-collector.slice
+      permissions: 600
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector.timer
+      destination: /etc/systemd/system/aks-log-collector.timer
+      permissions: 600
+    # aks check network
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-check-network.sh
+      destination: /opt/azure/containers/aks-check-network.sh
+      permissions: 755
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-check-network.service
+      destination: /etc/systemd/system/aks-check-network.service
+      permissions: 600
+    # aks logrotate
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-logrotate-override.conf
+      destination: /etc/systemd/system/logrotate.timer.d/override.conf
+      permissions: 644
+    # nftables
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/ipv6_nftables
+      destination: /etc/systemd/system/ipv6_nftables
+      permissions: 644
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/ipv6_nftables.service
+      destination: /etc/systemd/system/ipv6_nftables.service
+      permissions: 600
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/ipv6_nftables.sh
+      destination: /opt/scripts/ipv6_nftables.sh
+      permissions: 744
+    # kms
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/kms.service
+      destination: /etc/systemd/system/kms.service
+      permissions: 600
+    # containerd
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/containerd.service
+      destination: /etc/systemd/system/containerd.service
+      permissions: 600
+    # localdns
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/localdns.service
+      destination: /etc/systemd/system/localdns.service
+      permissions: 600
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/localdns.sh
+      destination: /opt/azure/containers/localdns.sh
+      permissions: 755
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/localdns-delegate.conf
```

Comment:
we also need `vhdbuilder/packer/prefetch.sh` (which is generated on the fly during the build before imagecustomerizer will be invoked) to land in `/opt/azure/containers/prefetch.sh`

Reply:
ah I found it, nevermind